### ISSUE TITLE
[WCAG 1.4.3]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.

### ISSUE THREAD

**Issue by rahulpundir019** [twitter/opensource-website#134]
**Describe the bug**
The color contrast ratio of the 'View all Projects' and 'View all Careers' buttons with respect to its background is less than required 4.5:1.

**To Reproduce**
1. Go to https://opensource.twitter.dev/
2. Run Accessibility Insights for Web 'FastPass' test on URL.
3. See error.

**Expected behavior**
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.

**Screenshots**
![image](https://user-images.githubusercontent.com/92663228/160240918-3ffb2a2a-25f9-4dc9-a6f7-d5fe781d51c2.png)

**Environment**
OS Windows 10  Version 20H2
Microsoft Edge version 99.0.1150.52
Tool: Accessibility Insights for Web

**How to fix:** 
Fix any of the following:
  Element has insufficient color contrast of 2.82 (foreground color: #ffffff, background color: #1da1f2, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1

**Additional context**
**User Experience**: People with low vision will find difficulty in identifying/reading the text/info as its color contrast ratio is low.

**WCAG Rule Reference**: https://www.w3.org/TR/WCAG21/#contrast-minimum

**AI Tool Reference**:This accessibility issue was found using Accessibility Insights for Web 2.31.1 (axe-core 4.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by rahulpundir019** [comment]
Issue is fixed now.
